tests/unittest/_torch/auto_deploy/unit/singlegpu/shim/test_engine.py

```diff
@@ -40,7 +40,9 @@ def get_inference_model(cache_seq_interface):
 
 
 @pytest.mark.parametrize("engine_cls", [ADEngine, DemoEngine])
-@pytest.mark.parametrize("attn_backend, attn_page_size", [("triton", 0), ("flashinfer", 2)])
+@pytest.mark.parametrize(
+    "attn_backend, attn_page_size", [("triton", 0), ("flashinfer", 2), ("torch", 0)]
```

Comment:
Let's also add a test to `tests/unittest/_torch/auto_deploy/unit/singlegpu/test_ad_build_small_single.py` with `torch` backend